src/test/java/org/jabref/logic/util/io/RegExpBasedFileFinderTests.java

```diff
@@ -56,7 +57,7 @@ public void testFindFiles() {
         RegExpBasedFileFinder fileFinder = new RegExpBasedFileFinder("**/[bibtexkey].*\\\\.[extension]", ',');
 
         //when
-        List<Path> result = fileFinder.findAssociatedFiles(localEntry, dirs, extensions);
+        Collection<Path> result = fileFinder.findAssociatedFiles(localEntry, dirs, extensions);
```

Comment:
Here you can change them back to list again

Reply:
Done!